source/includes/getting-started/_install.html.md

```diff
@@ -0,0 +1,181 @@
+```
+
+1. **Git**: Git is more than likely already installed on your computer if you have Xcode, but if not, run the git command or install with [Homebrew][6].
+2. **Python**: Download and install Python from [python.org][3] or with [Homebrew][6].
+3. **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+## Install with Anaconda
+
+If you *do not* have Anaconda installed on your system, please skip this section.
+
+conda is a tool for installing and managing Python and R dependencies. For more information, refer to the [Anaconda website][10].
+
+> Windows
+
+```shell
+conda create -n gym-env
+activate gym-env
+conda install pip
+```
+
+> Mac OS
+
+```shell
+conda create -n gym-env
+source activate gym-env
+conda install pip
+```
+
+# Setup the Bonsai CLI
+
+Install the Bonsai Command Line Interface tool and configure it. You'll need to have your access code (accessKey). You can find your access code in your [Bonsai Account Settings][8].
+
+```shell
+pip install bonsai-cli
+bonsai configure # Follow the instructions to retrieve your key, and enter it when prompted
+```
+
+## Clone a sample project
+
+Clone a sample simulation project with git. For this guide we'll be walking you through [OpenAI Gym's][4] Mountain Car simulation.
+
+```shell
+git clone https://github.com/BonsaiAI/gym-mountaincar-sample
+```
+
+## Install requirements
+
+Enter into the Mountain Car folder and then install the Mountain Car requirements from OpenAI Gym.
+
+```shell
+cd gym-mountaincar-sample
+pip install -r requirements.txt
+```
+
+# Create a BRAIN
+
+Create your BRAIN and give it a name. You can also create a BRAIN from your [beta.bons.ai][5] BRAIN Dashboard instead of using the command line. Either way, you will view your BRAIN's progress on the BRAIN Details page.
+
+### Create from Web
+
+![BRAINS Dashboard Page][15]
+
+The dashboard has a NEW BRAIN button, pictured above. Click on it to start the BRAIN creation process, where you're provided a form to give your brain a name and description. Name can contain alpha, numerals, or hyphens; the description is arbitrary and should contain human-readable text that explains your intent for the BRAIN.
+
+### Create from Command Line
+
+```shell
+bonsai create myMountainCarBrain
+```
+You may give you brain a name with `bonsai create brainName` but there is no option for a description with this method.
+
+# Train a BRAIN
+
+## Load sample Inkling into your BRAIN
+
+```shell
+bonsai load
+```
+
+Load your sample Inkling file for the Mountain Car simulation.
+
+## Start Training Mode
+
+```shell
+bonsai train start
+```
+
+Almost there! Time to tell the AI engine to prepare a new BRAIN version for training. Start the training mode for your BRAIN.
+
+## Connect the simulator for training
+
+```shell
+python mountaincar_simulator.py --train-brain=myMountainCarBrain --headless
```

Comment:
A user on Mac OS X could probably get away with installing virtualenv into their system python and then using a pure virtualenv install workflow.